tests/integration_tests/retrievers/test_azure_cognitive_search.py

```diff
@@ -0,0 +1,26 @@
+"""Test Azure Cognitive Search wrapper."""
+
+import os
+from typing import List
+
+from langchain.schema import Document
+from langchain.retrievers.azure_cognitive_search import AzureCognitiveSearchRetriever
+
+
+def test_azure_cognitive_search_call() -> None:
+    """Test valid call to Azure Cognitive Search."""
+
+    assert os.environ.get("TEST_AZURE_COGNITIVE_SEARCH_SERVICE") is not None
```

Comment:
not very familiar with ACS but if it's not uncommon to store search_service and search_index as env vars should we make it possible to retriever those params from env vars (the way we do with api keys?)

Reply:
I agree - done ✅